handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/BizkeeperHandler.java

```diff
@@ -72,8 +76,14 @@ public BizkeeperHandler(String groupname) {
   @Override
   public void handle(Invocation invocation, AsyncResponse asyncResp) {
     HystrixObservable<Response> command = delegate.createBizkeeperCommand(invocation);
-
     Observable<Response> observable = command.toObservable();
+    HystrixCircuitBreaker circuitBreaker =
+        HystrixCircuitBreaker.Factory.getInstance(CommandKey.toHystrixCommandKey(groupname, invocation));
+    if (circuitBreaker != null && circuitBreaker.isOpen()) {
+      EventManager.post(new CircutBreakerEvent(invocation, this.groupname, Type.OPEN));
+    }else {
+      EventManager.post(new CircutBreakerEvent(invocation, this.groupname, Type.CLOSE));
```

Comment:
We need only send event when last time is OPEN and turn to be CLOSED. Can't send for each invocation.